src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -283,11 +289,40 @@ impl Action for IngestAction {
                 Format::Protobuf {
                     message, schema, ..
                 } => {
+                    // TODO(chae): once this is fixed https://github.com/stepancheg/rust-protobuf/issues/576,
+                    // use protobuf-codegen-pure, an actual parser, rather than a regex.  For now, this is
+                    // just used in testdrive so it's not worth hand-rolling something for now.
+                    lazy_static! {
+                        static ref PROTO_IMPORT_REGEX: Regex =
+                            Regex::new(r#"import\s+["'](.*)["']"#).unwrap();
+                    }
                     if self.publish {
                         let schema = schema.expect("schema");
                         let ccsr_subject = format!("{}-{}", topic_name, typ);
+                        let schema_refs: Vec<_> = PROTO_IMPORT_REGEX
+                            .captures_iter(&schema)
+                            .filter_map(|capture| {
+                                let imp = capture
+                                    .get(1)
+                                    .expect("REGEX HAS ONE GROUP")
+                                    .as_str()
+                                    .to_owned();
+                                futures::executor::block_on(ccsr_client.get_subject(&imp))
```

Comment:
Eep! It is pretty much always wrong to use `block_on`. Easier thing to do here is probably to use a `for` loop instead so you can call `await`, or you can get fancy with the streams API if you want to fetch in parallel.

Reply:
Even for testdrive? I had imagined that's much more lax since it's not actually production code

Reply:
lol whoops I missed it's inside an async func.  my b

Reply:
> I had imagined that's much more lax since it's not actually production code

It generally is, but yeah, reasoning about how `futures::executor::block_on` works inside of an `async` function requires a PhD in Rust async runtimes.